src/library_fs.js

```diff
@@ -1136,6 +1171,15 @@ FS.staticInit();` +
       }
       stream.position = stream.stream_ops.llseek(stream, offset, whence);
       stream.ungotten = [];
+      #if FS_DEBUG
+      try {
+        if (stream.path && FS.trackingDelegate['onSeekFile']) {
+          FS.trackingDelegate['onSeekFile'](stream.path, stream.position, whence);
+        }
+      } catch(e) {
+        err("FS.trackingDelegate['onSeekFile']('"+stream.path+"', " + stream.position + ", " + whence + ") threw an exception: " + e.message);
```

Comment:
```suggestion
        err("FS.trackingDelegate['onSeekFile']('" + stream.path + "', " + stream.position + ", " + whence + ") threw an exception: " + e.message);
```
While we're changing this code, it would be good to make the existing spacing consistent as well.